### ISSUE TITLE
Not sure if this is an issue with my computer

### ISSUE THREAD

**Issue by zkg363n** [calthoff/self_taught#19]
Hello World! I'm new here!
I have the following issue... 
When i put the following 

![image](https://user-images.githubusercontent.com/38021247/38222627-e61c1398-36b3-11e8-9f0c-e7bea0dc52a7.png)

This on my end...
![image](https://user-images.githubusercontent.com/38021247/38222734-92fb237e-36b4-11e8-8915-e75be780d4f1.png)
it doesn't display the 100
no results

The book shows this...
![image](https://user-images.githubusercontent.com/38021247/38222775-ee523168-36b4-11e8-9a7e-899b5d45093f.png)

At the end it supposed to display 100 
Please advise.

**Comment by calthoff** [comment]
Hey there! When you type code into Python's shell, the output is sometimes different than when you save a program and run it. Try typing the same code into the shell, instead of saving and running it, and 100 should print.

**Comment by zkg363n** [comment]
Yes! That worked!  I thought I was doing something wrong. Thanks!

**Comment by GregKail** [comment]
I'm not sure what i'm missing, but when I save as and run any math do not get the expected result 
I get this
Python 3.7.2 (tags/v3.7.2:9a3ffc0492, Dec 23 2018, 22:20:52) [MSC v.1916 32 bit (Intel)] on win32
Type "help", "copyright", "credits" or "license()" for more information.
>>> 
============= RESTART: C:/Users/gkail/Desktop/length diagonal.py =============
>>>
